cln/cln.go

```diff
@@ -95,7 +89,20 @@ func (c *Cln) Connect() error {
 }
 
 func (c *Cln) Name() string {
-	return string(serviceName)
+	return "CLN"
+}
+
+func (c *Cln) GetWalletInfo() onchain.WalletInfo {
+	return c.walletInfo
+}
+
+func (c *Cln) SetId(id int64) {
+	c.walletInfo = onchain.WalletInfo{
```

Comment:
This way, `SetId` initializes the wallet. I don't like that naming. How about `SetupWallet`?